mailgun/resource_mailgun_domain.go

```diff
@@ -48,84 +49,28 @@ func resourceMailgunDomain() *schema.Resource {
 				ForceNew: true,
 				Optional: true,
 			},
-
-			"receiving_records": &schema.Schema{
-				Type:     schema.TypeList,
-				Computed: true,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"priority": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"record_type": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"valid": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"value": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-					},
-				},
-			},
-
-			"sending_records": &schema.Schema{
-				Type:     schema.TypeList,
-				Computed: true,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"name": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"record_type": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"valid": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-						"value": &schema.Schema{
-							Type:     schema.TypeString,
-							Computed: true,
-						},
-					},
-				},
-			},
```

Comment:
Removed these attributes because these cannot be set or updated by any of the APIs.